mesh/exodus_writer.cpp

```diff
@@ -0,0 +1,1000 @@
+// Copyright (c) 2010-2024, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "mesh_headers.hpp"
+#include <unordered_set>
+
+#ifdef MFEM_USE_NETCDF
+#include "netcdf.h"
+#endif
+
+/// @brief Call NetCDF functions inside the macro. This will provide basic
+/// error-handling.
+#define CHECK_NETCDF_CODE(return_code)\
+{\
+   if ((return_code) != NC_NOERR)\
+   {\
+      MFEM_ABORT("NetCDF error: " << nc_strerror((return_code)));\
+   }\
+}
+
+namespace mfem
+{
+
+#ifdef MFEM_USE_NETCDF
+
+// Variable labels
+const char * EXODUS_TITLE_LABEL = "title";
```

Comment:
We are reluctant to place variables directly into the `mfem` namespace. Even when they have such specific names as these. I think we would most often define these as `static` variables within something like the `ExodusIIWriter` class. If you prefer they could probably be placed within a separate utility class. Other possibilities exist. If you have a strong inclination towards a particular solution we can certainly discuss options.

Reply:
Okay, sure. How about plonking them into an "ExodusIILabels" namespace? I think this is a better solution than putting them into a utility class.

Reply:
As long as that namespace is inside the `mfem` namespace I think that would be fine.